content/en/observability_pipelines/guide/run_multiple_pipelines_on_the_same_host.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Run Multiple Pipelines on the Same Host
+disable_toc: false
+---
+
+## Overview
+
+If you want to run multiple pipelines on one host so that you can send logs from different sources, you need to manually add the Worker files for any additional Workers. This document goes over how to run additional Workers and the files that you need to add and modify to run them.
+
+## Prerequisites
+
+[Set up the first pipeline](https://docs.datadoghq.com/observability_pipelines/set_up_pipelines/?tab=pipelineui) and install the Worker on your host.
+
+## Create another pipeline
+
+[Set up another pipeline](https://docs.datadoghq.com/observability_pipelines/set_up_pipelines/?tab=pipelineui) for the additional Worker that you want to run on the same host. When you get to the Install page, follow the below steps to run the additional Worker.
```

Comment:
```suggestion
[Set up another pipeline](https://docs.datadoghq.com/observability_pipelines/set_up_pipelines/?tab=pipelineui) for each additional Worker you want to run on the same host. When you reach the Install page, follow the below steps to run the additional Worker.
```
Can it be one or more additional workers? If so suggest to change to "each additional" Worker.

Reply:
It's for one Worker, they would have go through these steps again to set up another one.